scripts/core/ui/components/video.tsx

```diff
@@ -20,8 +26,23 @@ export class VideoComponent extends React.PureComponent<IProps> {
             poster = item.renditions.viewImage.href;
         }
 
-        const videoRenditions = Object.values(item.renditions)
-            .filter((rendition) => rendition.mimetype != null && rendition.mimetype.startsWith('video'));
+        let videoRenditions: Array<IVideoRenditionItem> = [];
+
+        for (const rend of Object.values(item.renditions)) {
+            if (rend.mimetype == null) {
+                continue;
+            } else if (rend.mimetype.toLowerCase() === 'application/x-mpegurl') {
+                return <HLSVideoComponent
```

Comment:
> HLSVideoComponent can handle only one stream URL at a period of time

Can you add this as a comment here and in the component definition? This is a very important piece of knowledge that's not mentioned anywhere in the code.